File: BackEnd/Models/turno.cs
namespace BackEnd.Models
{
    public class turno
    {
        public int idTurno { get; set; }

        public int idServicio { get; set; }

        public DateTime fechaTurno { get; set; }

        public string horaInicio { get; set; }

        public string horaFin { get; set; }

        public bool estado { get; set; }
    }
}

File: src/app/turno/listar-turno/turno.ts
export interface turnoDTO{    
    idTurno: number,
    idServicio: number,
    fechaTurno : Date,
    horaInicio: string,
    horaFin: number,
    estado: boolean
}

export interface turnoCreacionDTO{        
    idServicio: number,
    fechaInicio: Date,
    fechaFin: Date
}

